app/src/main/java/com/codesoom/assignment/App.java

```diff
@@ -1,10 +1,16 @@
 package com.codesoom.assignment;
 
-public class App {
-    public String getGreeting() {
-        return "Hello World!";
-    }
+import com.codesoom.assignment.handler.RequestHandler;
+import com.sun.net.httpserver.HttpServer;
+import java.io.IOException;
+import java.net.InetSocketAddress;
 
-    public static void main(String[] args) {
+public class App {
+    public static void main(String[] args) throws IOException {
+        HttpServer httpServer = HttpServer.create();
+        InetSocketAddress inetSocketAddress = new InetSocketAddress("localhost", 8000);
+        httpServer.bind(inetSocketAddress,0);
```

Comment:
```suggestion
        httpServer.bind(inetSocketAddress, 0);
```

IDEA에 자동 정렬 기능을 사용해서 코드를 정렬하면 좋을 것 같아요. 습관적으로 실행하면 좋습니다.

Reply:
네! 습관적으로 실행하겠습니다.